x/consensus/depinject.go

```diff
@@ -38,11 +38,14 @@ type ModuleInputs struct {
 type ModuleOutputs struct {
 	depinject.Out
 
+	Authority     Authority
```

Comment:
why is this needed?

Reply:
it used to validate messages

Reply:
shouldn't we expose this from the consensus keeper instead? this feels a bit weird like this imho.